website/app/docs/page.mdx

```diff
@@ -4,16 +4,19 @@ export const metadata = {
 
 # Introduction
 
-Kardinal is a framework for creating extremely lightweight ephemeral development environments within a shared Kubernetes cluster.
-
-In Kardinal, an environment is called a "[flow](concepts/flows)" because it represents a path that a request takes through the cluster. Versions of services that are under development are deployed on-demand, and then shared across all development work that depends on that version.
-
-When you create a [flow](concepts/flows) to test a feature, Kardinal will deploy only the set of services that are changing for that feature. Then, any requests related to testing that feature are routed to those versions.
-
-As you onboard deeper into Kardinal, you will be able to set up isolated state for [flows](concepts/flows) when desired (i.e. for testing database migrations or write-intensive workloads on shared state).
-
-Even with isolated state per [flow](concepts/flows), Kardinal will still deploy the absolute minimum resources necessary to test the changes. Isolation is done at the level of the request route, not by duplicating services in your cluster unnecessarily.
-
-To see how Kardinal compares to other tools, and to get an idea of how would fit into your workflow, check out the [comparisons to alternatives](references/comparisons) section.
-
-To get started, [install Kardinal](getting-started/install) and then [create your first flow](getting-started/own-app).
\ No newline at end of file
+Kardinal is a framework for creating extremely lightweight development environments within a shared Kubernetes cluster that gives you the following benefits:
```

Comment:
I would retain

```
In Kardinal, an environment is called a "[flow](concepts/flows)" because it represents a path that a request takes through the cluster. Versions of services that are under development are deployed on-demand, and then shared across all development work that depends on that version.
```

before the next line, because we use the word "flow" later but its never introduced. have had that end up in confusion quite a bit - this should probably apply to the readme too

Reply:
Done 👍